Solutions/IronNet IronDefense/Playbooks/IronNet_UpdateSentinelIncidents/azuredeploy.json

```diff
@@ -2,67 +2,28 @@
   "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "metadata": {
-    "comments": "Enrich IronNet Alerts based on the corresponding Sentinel Incident updates via IronNet Endpoints",
+    "comments": "Enrich IronNet Alerts based on the corresponding Microsoft Sentinel Incident updates via IronNet Endpoints",
     "author": "IronNet"
   },
   "parameters": {
-    "IronApiUsername": {
-      "type": "String",
+    "PlaybookName": {
+      "type": "string",
+      "repeat": "false",
+      "defaultValue":"UpdateAzureSentinelIncidents",
       "metadata": {
-        "description": "Username used to authenticate to IronAPI"
-      }
-    },
-    "IronApiPassword": {
-      "type": "SecureString",
-      "metadata": {
-        "description": "Password used to authenticate to IronAPI"
-      }
-    },
-    "IronDefenseUrl": {
-      "type": "String",
-      "metadata": {
-        "description": "IronNet URL used to authenticate to IronAPI"
-      }
-    },
-    "ClientId": {
-      "type": "String",
-      "metadata": {
-        "description": "Azure Active Directory ClientId used to connect to the Azure Sentinel API to fetch Incidents"
-      }
-    },
-    "ClientSecret": {
-      "type": "SecureString",
-      "metadata": {
-        "description": "Azure Active Directory ClientSecret used to connect to the Azure Sentinel API to fetch Incidents"
-      }
-    },
-    "TenantId": {
-      "type": "String",
-      "metadata": {
-        "description": "Azure Active Directory TenantId of Azure Sentinel"
-      }
-    },
-    "WorkspaceName": {
-      "type": "String",
-      "metadata": {
-        "description": "Log Analytics Workspace connected to Azure Sentinel"
-      }
-    },
-    "Playbook Name": {
-      "type": "String",
-      "metadata": {
-        "description": "Playbook name to test the IronNet API"
-      }
+				"label": "Playbook Name",
+				"description": "Playbook name to test the IronNet API"
+			}
```

Comment:
Please add IronDefenseUrl, IronApiUsername, IronApiPassword, AzureClientId, AzureClientSecret, workspace in paramters section